tests/test_orchestrator.py

```diff
@@ -31,6 +31,8 @@ class TestOrchestrator(unittest.TestCase):
     reference_fasta = os.path.join(test_sub_dir, 'genome.fa')
     metadata_json = os.path.join(test_sub_dir, 'sub_metadata.json')
     metadata_xlsx = os.path.join(test_sub_dir, 'sub_metadata.xlsx')
+    metadata_xlsx_version = '3.0.0'
```

Comment:
We should also include a test for v2.x (probably don't need to replicate all of them, just one or two important ones)